apps/site/assets/js/support-form.js

```diff
@@ -115,7 +115,9 @@ function setupPhotoPreviews($, toUpload) {
   const $container = $(".photo-preview-container");
   $("#photo").change(function() {
     if (this.files.length >= 1) {
-      resizeAndHandleUploadedFile($, this.files[0], $container, toUpload);
+      Array.from(this.files).forEach(file =>
```

Comment:
`this.files` actually turns out to be an instance of [`FormData`](https://developer.mozilla.org/en-US/docs/Web/API/FormData) rather than a normal array.

Reply:
Looking through the typing of `FormData` I can't find any documented support for the property length.  Since it appears to be working I am thinking it doesn't need to be changed.  Though I am wondering if there is another way to write the `if (this.files.length)` to use the defined properties of the `FormData` type.

Reply:
For the record the old version worked fine but only for the first image. Because it only called `resizeAndHandleUploadedFile` for `this.files[0]`. But if the user selects multiple images we want to show all of them rather than just the first.
That is kinda funny, I can't find any documentation for `FormData` property length too... I can update this with a maybe more conventional approach.